src/app/events/page.tsx

```diff
@@ -106,7 +106,7 @@ type Props = {
   searchParams: NextServerSearchParams
 }
 
-const POSTS_PER_PAGE = 20
+const EVENTS_PER_PAGE = 20
```

Comment:
Love the title of this commit. And I love how you pick up these little initiatives. 🥰